src/lib/Scenes/Collection/Components/__tests__/__snapshots__/FeaturedArtists-tests.tsx.snap

```diff
@@ -1,3 +1,3 @@
tyle=\\"width: 45px; height: 45px; border-radius: 22.5px;\\" source=\\"[object Object]\\"></image></view><view justifycontent=\\"center\\" width=\\"0\\" flexgrow=\\"1\\"><text font-family=\\"ReactNativeAGaramondPro-Regular\\" font-size=\\"16px\\" lineheight=\\"20px\\" ellipsizemode=\\"tail\\" numberoflines=\\"1\\" color=\\"black100\\">Andy Warhol</text><text font-family=\\"Unica77LL-Regular\\" font-size=\\"12px\\" lineheight=\\"16px\\" ellipsizemode=\\"tail\\" numberoflines=\\"1\\" color=\\"black60\\">American, 1947-1987</text></view><view ml=\\"1\\" flexdirection=\\"row\\" alignitems=\\"center\\" justifycontent=\\"flex-end\\"><view flexdirection=\\"row\\" focusable=\\"true\\"><view variant=\\"secondaryOutline\\" theme=\\"[object Object]\\" px=\\"1\\"><view><text font-family=\\"Unica77LL-Medium\\" font-size=\\"12px\\" lineheight=\\"16px\\" color=\\"#000\\">Following</text></view><text font-family=\\"Unica77LL-Medium\\" font-size=\\"12px\\" lineheight=\\"16px\\" role=\\"presentation\\">Following</text></view></view></view></view></view><view width=\\"100%\\" pb=\\"20\\"><view flexdirection=\\"row\\" justifycontent=\\"space-between\\" flexwrap=\\"nowrap\\" focusable=\\"true\\"><view mr=\\"1\\" justifycontent=\\"center\\"><image resizemode=\\"stretch\\" style=\\"width: 45px; height: 45px; border-radius: 22.5px;\\" source=\\"[object Object]\\"></image></view><view justifycontent=\\"center\\" width=\\"0\\" flexgrow=\\"1\\"><text font-family=\\"ReactNativeAGaramondPro-Regular\\" font-size=\\"16px\\" lineheight=\\"20px\\" ellipsizemode=\\"tail\\" numberoflines=\\"1\\" color=\\"black100\\">Joan Miro</text><text font-family=\\"Unica77LL-Regular\\" font-size=\\"12px\\" lineheight=\\"16px\\" ellipsizemode=\\"tail\\" numberoflines=\\"1\\" color=\\"black60\\">Spanish, 1877-1983</text></view><view ml=\\"1\\" flexdirection=\\"row\\" alignitems=\\"center\\" justifycontent=\\"flex-end\\"><view flexdirection=\\"row\\" focusable=\\"true\\"><view variant=\\"secondaryOutline\\" theme=\\"[object Object]\\" px=\\"1\\"><view><text font-family=\\"Unica77LL-Medium\\" font-size=\\"12px\\" lineheight=\\"16px\\" color=\\"#000\\">Following</text></view><text font-family=\\"Unica77LL-Medium\\" font-size=\\"12px\\" lineheight=\\"16px\\" role=\\"presentation\\">Following</text></view></view></view></view></view><view focusable=\\"true\\"><view flexdirection=\\"row\\" justifycontent=\\"space-between\\" flexwrap=\\"nowrap\\"><view mr=\\"1\\" justifycontent=\\"center\\"><view width=\\"45px\\" height=\\"45px\\" justifycontent=\\"center\\" alignitems=\\"center\\"><text font-family=\\"ReactNativeAGaramondPro-Semibold\\" font-size=\\"16px\\" lineheight=\\"45\\" color=\\"black60\\">+ 2</text></view></view><view justifycontent=\\"center\\" width=\\"0\\" flexgrow=\\"1\\"><text font-family=\\"ReactNativeAGaramondPro-Regular\\" font-size=\\"16px\\" lineheight=\\"20px\\" ellipsizemode=\\"tail\\" numberoflines=\\"1\\" color=\\"black100\\">View more</text></view></view></view></view></view>"`;
```

Comment:
I would have expected his to be pretty formatted. Do we know if this is a consequence of using RN or something?

I mean, I go back and forth on the utility of snapshots… but this format _really_ seems to inhibit code review, which should ostensibly be part of the snapshot workflow, eh?

Reply:
I _think_ it's not formatted because it's comparing a *string*, not JSX (note the `"` at the beginning and end of the value). This might mean we just need to change the test code a bit to not convert it to a string - I'll dig a bit.

Reply:
`html()` which we call to do the comparison definitely returns a string. I tried just comparing just the tree object but that returns a ReactWrapper {} which isn't really helpful.

Reply:
Ok, thanks for looking into it!